### ISSUE TITLE
[1.2.3-1081-g766ba2a~utopic] Unknown desktop icon application-x-clementine in clementine.desktop

### ISSUE THREAD

**Issue by marcelpaulo** [clementine-player/Clementine#4819]
`clementine.desktop` is using unknown desktop icon `application-x-clementine`:

```
paulo@monk:~$ grep -i icon /usr/share/applications/clementine.desktop 
Icon=application-x-clementine
```

Icon `application-x-clementine` is not installed:

```
paulo@monk:~$ find /usr/share/icons -name '*clementine*'
/usr/share/icons/ubuntu-mono-light/apps/24/clementine-panel-grey.png
/usr/share/icons/ubuntu-mono-light/apps/24/clementine-panel.png
/usr/share/icons/hicolor/64x64/apps/clementine.png
/usr/share/icons/hicolor/128x128/apps/clementine.png
/usr/share/icons/hicolor/scalable/apps/clementine.svg
/usr/share/icons/ubuntu-mono-dark/apps/24/clementine-panel-grey.png
/usr/share/icons/ubuntu-mono-dark/apps/24/clementine-panel.png
```

I'm running clementine **1.2.3-1081-g766ba2a~utopic** from PPA **ppa:me-davidsansome/clementine-dev**, on Xubuntu 14.10

**Comment by ArnaudBienner** [comment]
Very likely to be related to commit 26718c777ae854a68ed45b1a8f9e0929b156fd90

**Comment by Chocobozzz** [comment]
Yes, I forgot to update *dist/clementine.desktop* line 31. @ArnaudBienner Can you fix this for me ?

**Comment by Chocobozzz** [comment]
Finally made another PR

**Comment by hatstand** [comment]
Fixed by a07ff9845693a72b5313ea47ca6a6e8af06c4827
